rio_cogeo/cogeo.py

```diff
@@ -135,6 +131,8 @@ def cog_translate(  # noqa: C901
         Use GDAL COG driver if set to True. COG driver is available starting with GDAL 3.1.
 
     """
+    out_profile = dst_kwargs.copy()
```

Comment:
because we edit the profile it's safer to create a copy

Reply:
My nit would be: any reason not to do
```py
dst_kwargs = dst_kwargs.copy()
```
? Aka any reason you need to hold on to the original `dst_kwargs` reference? If not, this PR's diff will be much smaller.